### ISSUE TITLE
smaller learning rate not applied during finetuning?

### ISSUE THREAD

**Issue by mdukaczewski** [naoto0804/pytorch-inpainting-with-partial-conv#9]
https://github.com/naoto0804/pytorch-inpainting-with-partial-conv/blob/d1a29ff7a021fbc9e6e37b75b1b5fceab202dcb9/train.py#L93

I don't think you are applyng lr parameter to your optimizer

**Comment by naoto0804** [comment]
Right, I will fix it, thank you!
